package.json

```diff
@@ -5,9 +5,19 @@
   },
   "dependencies": {
     "@rails/webpacker": "3.5",
+    "@types/react": "^16.4.7",
+    "@types/react-dom": "^16.0.6",
+    "babel-preset-react": "^6.24.1",
+    "brave-ui": "^0.23.0",
     "clipboard": "^1.7.1",
     "chart.js": "^2.7.2",
-    "rails-ujs": "^5.1.4"
+    "prop-types": "^15.6.2",
+    "rails-ujs": "^5.1.4",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "styled-components": "^3.2.6",
+    "ts-loader": "^3.5.0",
```

Comment:
Not sure if you saw my comment on the previous but we should lock this down because webpacker doesn't support webpack 4 in the current release, and they introduced this commit to lock it down
rails/webpacker@af471dc#diff-63cad75e7b69369d47bacd65341a3bf8

Reply:
Hey Cory, sorry I didn't see your comment.

@diracdeltas FYI, if it's fine, I'm going to keep this locked at current version until it's resolved.

Reply:
locking temporarily sounds fine